main/unbound/unbound.initd

```diff
@@ -22,36 +25,20 @@ depend() {
 	after auth-dns
 }
 
-checkconfig() {
-	checkpath --directory /var/run/unbound
-	UNBOUND_PIDFILE=$("${UNBOUND_CHECKCONF}" -o pidfile "${UNBOUND_CONFFILE}")
-	return $?
-}
-
 configtest() {
-	ebegin "Checking ${SVCNAME} configuration"
-	checkconfig
+	ebegin "Checking $RC_SVCNAME configuration"
+	pidfile=$("$UNBOUND_CHECKCONF" -o pidfile "$UNBOUND_CONFFILE")
```

Comment:
pidfile location should be defined in runscript, not in the program’s configuration file. It’s init/rc’s system responsibility to create and maintain pidfile (if it needs them).